src/pages/react/navigation.md

```diff
@@ -117,12 +116,12 @@ const DashboardPage: React.FC<RouteComponentProps> = ({match}) => {
 };
 ```
 
-Here, we see that in the event a location does not match the first two `Route`s it will redirect the Ionic React app to the `match.url` path. 
+Here, we see that in the event a location does not match the first two `Route`s. The `IonRouterOutlet` will redirect the Ionic React app to the `match.url` path.
```

Comment:
This first part should be reworded because it is not a complete sentence.

Reply:
D'oh! Let me fix it 😄